lessons/22-js-conditionals/index.js

```diff
@@ -0,0 +1,16 @@
+alert('Enter two numbers to see if first greater then second');
+let firstNumber= prompt("Enter first number: ");
+let secondNumber= prompt("Enter second number:");
+let result=firstNumber>secondNumber;
+if (result) {
+    alert("firstNumber > secondNumber")
+    
+} else {
+    if (firstNumber===secondNumber) {
+        alert("firstNumber is equal to secondNumber")
+        
+    } else {
+        alert("firstNumber < secondNumber")
+    }
+    
+}
```

Comment:
This works, but try and refactor it to use if / else if / else all in one line :)